OTKit/otkit-icons/ic_anniversary_fill.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 51.1 (57501) - http://www.bohemiancoding.com/sketch -->
+    <title>Icon/Fill/ic_anniversary_fill</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Icon/Fill/ic_anniversary_fill" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M8.5,12.8064227 L8.5,10.5 L5.5,10.5 L5.5,13.8064227 L8.5,12.8064227 Z M6,17.8292943 C4.83480763,17.4174579 4,16.3062188 4,15 L4,9.5 C4,9.22385763 4.22385763,9 4.5,9 L9.5,9 C9.77614237,9 10,9.22385763 10,9.5 L10,15 C10,16.3062188 9.16519237,17.4174579 8,17.8292943 L8,20 C8.27614237,20 8.5,20.2238576 8.5,20.5 L8.5,21.5 C8.5,21.7761424 8.27614237,22 8,22 L6,22 C5.72385763,22 5.5,21.7761424 5.5,21.5 L5.5,20.5 C5.5,20.2238576 5.72385763,20 6,20 L6,17.8292943 Z M14.903591,4.99071337 C14.6735802,4.94578389 14.5,4.74316286 14.5,4.5 L14.5,2.5 C14.5,2.22385763 14.7238576,2 15,2 L17,2 C17.2761424,2 17.5,2.22385763 17.5,2.5 L17.5,4.5 C17.5,4.74856702 17.3186186,4.95477013 17.0809911,4.99347402 C17.3017452,6.7932474 17.584275,8.12875582 17.9285715,9 C18.1560977,9.57575641 18.4183085,10.1580761 18.7152038,10.7469592 C19.559955,12.4224902 20,14.2727764 20,16.1492115 L20,21.0001314 C20,21.5524161 19.5522847,22.0001314 19,22.0001314 L12.9999941,22.000132 C12.4477093,22.000132 11.9999941,21.5524168 11.9999941,21.000132 L11.9999962,16.1384461 C11.9999974,14.2729494 12.43493,12.4331438 13.270258,10.7651195 C13.5682511,10.1700723 13.8298803,9.58169911 14.0551456,9 C14.3927375,8.12824216 14.6755526,6.79181328 14.903591,4.99071337 Z M16.5547002,13.3698001 C16.2188008,13.1458672 15.7811992,13.1458672 15.4452998,13.3698001 L13.6240377,14.5839749 C13.5691116,14.6205923 13.5219801,14.6677238 13.4853627,14.7226499 C13.3321864,14.9524142 13.3942734,15.2628489 13.6240377,15.4160251 L15.4452998,16.6301999 C15.7811992,16.8541328 16.2188008,16.8541328 16.5547002,16.6301999 L18.3759623,15.4160251 C18.4308884,15.3794077 18.4780199,15.3322762 18.5146373,15.2773501 C18.6678136,15.0475858 18.6057266,14.7371511 18.3759623,14.5839749 L16.5547002,13.3698001 Z" id="ic_anniversary_Fill" fill="#2D333F"></path>
```

Comment:
Is `id="ic_anniversary_Fill"` correct?
(notice the capital `F`)

Reply:
if that doesn't impact dev side it's fine. I can also update and re export easily with a lowercase f